ci/release/update-version.sh

```diff
@@ -60,6 +60,8 @@ for FILE in conda/environments/*.yml; do
    sed_runner "s/rmm=${CURRENT_SHORT_TAG}/rmm=${NEXT_SHORT_TAG}/g" ${FILE};
    sed_runner "s/libraft-headers=${CURRENT_SHORT_TAG}/libraft-headers=${NEXT_SHORT_TAG}/g" ${FILE};
    sed_runner "s/pyraft=${CURRENT_SHORT_TAG}/pyraft=${NEXT_SHORT_TAG}/g" ${FILE};
+   sed_runner "s/raft-dask=${CURRENT_SHORT_TAG}/libraft-headers=${NEXT_SHORT_TAG}/g" ${FILE};
```

Comment:
Are you intentionally rewriting `raft-dask` to `libraft-headers` or is that a copy-paste error?

Reply:
Thanks for the catch, it was a copy-paste error.